MathlibTest/FindSyntax.lean

```diff
@@ -57,22 +62,26 @@ info: Found 1 use among over 500 syntax declarations
 info: Found 4 uses among over 500 syntax declarations
 ---
 
-Lean.Parser.Tactic.refine':
+Lean.Parser.Tactic.refine': defined in 'Init.Tactics'
+start: ⟨212, 0⟩, end: ⟨216, 49⟩
```

Comment:
Another reason not to include these numbers: these tests are going to be very fragile with respect to Lean updates